src/Raven.Client/Documents/Session/IAbstractDocumentQuery.cs

```diff
@@ -149,16 +149,12 @@ public interface IAbstractDocumentQuery<T>
         /// <summary>
         ///   Matches fields which starts with the specified value.
         /// </summary>
-        /// <param name = "fieldName">Name of the field.</param>
-        /// <param name = "value">The value.</param>
-        void WhereStartsWith(string fieldName, object value);
+        void WhereStartsWith(string fieldName, object value, bool exact = false);
```

Comment:
This is a breaking change. Better to create an overload here.

Reply:
We guarantee source-level compatibility between minors, so it should be OK?